platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java

```diff
@@ -66,4 +67,38 @@ public static void logNotKnownConfigProperties(@NonNull final Configuration conf
                     logger.warn(STARTUP.getMarker(), message);
                 });
     }
+
+    /**
+     * Logs all applied mapped properties. And suggests to change the new property name.
+     */
+    private static void logAppliedMappedProperties(@NonNull final Set<String> configNames) {
+        final Map<String, String> mappings = ConfigMappings.MAPPINGS.stream()
+                .collect(Collectors.toMap(ConfigMapping::originalName, ConfigMapping::mappedName));
+
+        configNames.stream().filter(mappings::containsKey).forEach(name -> {
+            final String message = ("Configuration property '%s' was mapped to '%s'. "
+                            + "Consider change the new property name")
```

Comment:
```suggestion
            final String message = ("Configuration property '%s' was renamed to '%s'. "
                            + "This build is currently backwards compatible with the old name, but this may not be true in a future release, so it is important to switch to the new name.")
```

Suggestion for the warning message (although you probably need to fix line wrapping from my suggestion).

Reply:
Done, thanks!